### ISSUE TITLE
Tooltip on multiple fields always show on last field

### ISSUE THREAD

**Issue by Rias500** [formvalidation/support#233]
When validating multiple fields, if you set the container option as 'tooltip' the tooltip that shows up on focus will always be on the last field:

JSFiddle with the *require at least one field* example edited to use tooltips:
http://jsfiddle.net/gghg6thh/

![image](https://cloud.githubusercontent.com/assets/3626559/8958638/58f59daa-3606-11e5-986b-7e8ac92ec43c.png)

I've tracked this down to the _initField function lines 409-419:

```
$field
    // Show tooltip/popover message when field gets focus
    .off('focus.container.' + ns)
    .on('focus.container.' + ns, function() {
        that._showTooltip($field, container);
    })
    // and hide them when losing focus
    .off('blur.container.' + ns)
    .on('blur.container.' + ns, function() {
        that._hideTooltip($field, container);
    });
```

I implemented a temporary fix by using $(this) instead of $field inside the events which makes the tooltips show up at the correct field, but maybe you know a better solution:

```
$field
    // Show tooltip/popover message when field gets focus
    .off('focus.container.' + ns)
    .on('focus.container.' + ns, function() {
        that._showTooltip($(this), container);
    })
    // and hide them when losing focus
    .off('blur.container.' + ns)
    .on('blur.container.' + ns, function() {
        that._hideTooltip($(this), container);
    });
```

**Comment by nghuuphuoc** [comment]
Thank for reporting this. Your fix looks good :+1: 
This will be fixed for the next v0.7.0 version.

**Comment by nghuuphuoc** [comment]
Done. 
[v0.7.0](http://formvalidation.io/news/v0.7.0-released/) is released. You can use the previous download link to get new version.
